verification/cbmc/sources/openssl/ec_override.c

```diff
@@ -488,5 +488,11 @@ void write_unconstrained_data(unsigned char *out, size_t len) {
 
     // Currently we ignore the len parameter and just fill the entire buffer with unconstrained data.
     // This is fine because it is strictly more general behavior than writing only len bytes.
-    __CPROVER_havoc_object(out);
+    // __CPROVER_havoc_object(out);
+    if (len > 0) {
+        size_t index;
+        unsigned char nondet_char;
+        __CPROVER_assume(index < len);
+        out[index] = nondet_char;
+    }
```

Comment:
Do we have a GitHub issue to track this issue? Both here and in our openssl verification model?

Reply:
Yes, we do: #653 https://github.com/awslabs/aws-verification-model-for-libcrypto/issues/18